scripts/normalize-pbxproj.py

```
@@ -0,0 +1,150 @@
+    """
+    comment = ENTRY_COMMENT_RE.search(line)
+    label = comment.group("label").lower() if comment else line.strip().lower()
+    uuid = line.lstrip().split(" ", 1)[0]
+    return (label, uuid)
+
+
+def sort_flat_section(lines: list[str], section: str) -> list[str]:
+    begin = f"/* Begin {section} section */"
+    end = f"/* End {section} section */"
+    try:
+        start = next(i for i, l in enumerate(lines) if l.strip() == begin)
```

Ruff (0.15.14)

[error] Ambiguous variable name: `l`

(E741)